lib/design/app_theme.dart

```diff
@@ -92,34 +92,8 @@ extension CustomStyles on TextTheme {
   TextStyle get subtitle2LowEmphasis =>
       Typography.material2018().englishLike.subtitle2.copyWith(fontSize: 14, fontWeight: FontWeight.w300);
 
-  TextStyle get subtitle2OpacityEmphasis => Typography.material2018()
```

Comment:
????? Why are you making changes to the text themes? 

I think you need to merge master into your branch.

Reply:
weird I have master latest code, maybe was an error when I merge.